src/sass/components/_repo-images.scss

```diff
@@ -0,0 +1,42 @@
+// Repo Images
+
+// Variables specific to repo images
+$repo-img-size: calc(3.5rem + 3vw);
+$repo-img-size-static: 4.75rem;
+$repo-img-pos-adjust: -1.125rem;
+
+.ms-repo-item {
+  // increase default height/weight of icon in repo visualpicker
+  .slds-media__figure {
+    > div {
+      height: $repo-img-size;
+      width: $repo-img-size;
+
+      .slds-avatar_large {
+        height: 100%;
+        width: 100%;
+        font-size: $font-size-10;
+        font-weight: $font-weight-bold;
+      }
+
+      abbr {
+        text-shadow: none;
+      }
+    }
+  }
+}
+
+// When image, name and description are displayed on project list sidebar
+.ms-repo-image-wrapper {
+  height: $repo-img-size;
+  width: $repo-img-size;
+}
+
+// When image and name are displayed in header
+.ms-repo-image-header {
+  height: $repo-img-size-static;
+  // .slds-page-header__icon is 2.25rem -- account for vertical space
+  margin-bottom: $repo-img-pos-adjust;
+  margin-top: $repo-img-pos-adjust;
```

Comment:
would flexbox vertical centering work here, rather than explicit negative margins?

Reply:
the negative margin helps make sure the global page header's height stays the same, since the header has an x-large padding all around ... it basically allows this image to flow outside it's area without effecting the other things in the page header.